src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/SourceInputStreamFactory.java

```diff
@@ -152,7 +152,7 @@ public MessageHandler newStream()
         private final Int2ObjectHashMap<Http2Stream> http2Streams;
         private int lastPromisedStreamId = 0;
 
-        private final AtomicBuffer slab = new UnsafeBuffer(new byte[4096]);
+        private AtomicBuffer slab;
```

Comment:
The type of this field should be `MutableDirectBuffer` if we're not using any of the `AtomicBuffer` specific methods.